chromadb/test/property/test_cross_version_persist.py

```diff
@@ -256,18 +256,24 @@ def persist_generated_data_with_old_version(
             # In order to test old versions, we can't rely on the not_implemented function
             embedding_function=not_implemented_ef(),
         )
-        coll.add(**embeddings_strategy)
+        result = coll.add(**embeddings_strategy)
```

Comment:
- Old versions will not return a `result` 
- Old versions can't accept a `RecordSet` with null IDs. 

We shold determine which version we are in when `add` is called and do the right thing accordingly.

Reply:
I'm not sure how this test is currently passing.